README.md

```diff
@@ -150,15 +150,15 @@ Ghostfolio is available for various home server systems, including [CasaOS](http
 
 - [Docker](https://www.docker.com/products/docker-desktop)
 - [Node.js](https://nodejs.org/en/download) (version 20+)
-- [Yarn](https://yarnpkg.com/en/docs/install)
+- [npm](https://docs.npmjs.com/cli/v10/commands/npm-install)
```

Comment:
Is `npm` explicitly needed or is it already part of `Node.js`?

Reply:
I'm sorry I thought of including nvm, but added npm instead,my bad. Yes, it's already part of nodejs.